resources/lang/tha.ini

```diff
@@ -0,0 +1,112 @@
+; session
+session.undo.none = "ไม่มีอะไรให้ย้อนกลับ"
+session.undo.left = "คุณมีคำสั่งย้อนกลับเหลือ {%0}"
+session.redo.none = "ไม่มีอะไรให้ทำใหม่"
+session.redo.left = "คุณมีคำสั่งให้ทำใหม่เหลือ {%0}"
+session.brush.added = "เพิ่ม {%0} เข้าเซสชั่น"
+session.brush.deleted = "ลบ {%0} (UUID {%1})"
+session.brush.removed = "นำออก {%0} (UUID {%1})"
+session.language.set = "เปลี่ยนภาษาเป็น {%0} เรียบร้อย"
+session.language.notfound = "ไม่พบภาษา {%0} ปรับกลับเป็นค่าเริ่มต้น"
+; task
+task.copy.success = "คัดลอกสำเร็จ, ใช้เวลา {%0}, คัดลอก {%1} บล็อกจากทั้งหมด {%2}"
+task.count.success = "วิเคราะห์สำเร็จ, ใช้เวลา {%0}"
+task.count.result = "พบ {%0} บล็อกจากทั้งหมด {%1} บล็อก"
+task.fill.success = "เติมเต็มสำเร็จ, ใช้เวลา {%0}, เปลี่ยน {%1} บล็อกจากทั้งหมด {%2}"
+task.replace.success = "วางทับสำเร็จ ใช้เวลา {%0}, เปลี่ยน {%1} บล็อกจากทั้งหมด {%2}"
+task.revert.undo.success = "ย้อนกลับสำเร็จ, ใช้เวลา {%0}, เปลี่ยน {%1} บล็อกจากทั้งหมด {%2}"
+task.revert.redo.success = "ทำใหม่สำเร็จ, ใช้เวลา {%0}, เปลี่ยน {%1} บล็อกจากทั้งหมด {%2}"
+; flags
+flags.keepexistingblocks = "เก็บบล็อกที่มีอยู่ไว้"
+flags.keepair = "เก็บอากาศ"
+flags.hollow = "โพรง"
+flags.hollowclosed = "โพรงที่มีทางออกปิด"
+flags.natural = "ตามธรรมชาติ"
+; tools
+; wand tool
+tool.wand = "คทา"
+tool.wand.lore.1 = "คลิกซ้ายที่บล็อกเพื่อตั้งตำแหน่งที่ 1 ของการเลือก"
+tool.wand.lore.2 = "คลิกขวาที่บล็อกเพื่อตั้งตำแหน่งที่ 2 ของการเลือก"
+tool.wand.lore.3 = "ใช้คำสั่ง //togglewand เพื่อสลับการทำงานของคทา"
+tool.wand.disabled = "คทาถูกปิดใช้งาน ใช้คำสั่ง //togglewand เพื่อเปิดใช้งาน"
+tool.wand.setenabled = "คทาได้ถูก{%0}!"
+; debug tool
+tool.debug = "เครื่องมือ Debug"
+tool.debug.lore.1 = "คลิกซ้ายที่บล็อกเพื่อดูข้อมูล"
+tool.debug.lore.2 = "เช่นดูชื่อหรือค่าความเสียหายของบล็อก"
+tool.debug.lore.3 = "ใช้คำสั่ง //toggledebug เพื่อสลับการทำงานของเครื่องมือ"
+tool.debug.disabled = "เครื่องมือ Debug ถูกปิดใช้งาน ใช้คำสั่ง //toggledebug เพื่อเปิดใช้งาน"
+tool.debug.setenabled = "เครื่องมือ Debug ถูก{%0}!"
+; flood tool
+ui.flood.title = "เมนู Flood"
+ui.flood.options.limit = "จำนวนบล็อกที่มากที่สุด"
+ui.flood.options.blocks = "บล็อก"
+ui.flood.options.blocks.placeholder = "บล็อกที่แยกด้วย ;"
+ui.flood.options.label.infoapply = "กดปุ่ม "Submit" เพื่อนำมาใช้"
+; brush tool
+ui.brush.title = "เมนูแปรง"
+ui.brush.content = "เมนูแปรงหลัก"
+ui.brush.create = "สร้างใหม่"
+ui.brush.getsession = "เรียกเซสชั่นแปรง"
+ui.brush.edithand = "แก้ไขแปรงในมือ"
+; brush settings
+ui.brush.settings.title = "การตั้งค่าแปรง {%0}"
+; brush options
+ui.brush.options.blocks = "บล็อก"
+ui.brush.options.blocks.placeholder = "ตัวอย่าง: 1:1,2,tnt,log:12"
+ui.brush.options.diameter = "เส้นผ่าศูนย์กลาง"
+ui.brush.options.width = "กว้าง"
+ui.brush.options.height = "สูง"
+ui.brush.options.depth = "ลึก"
+ui.brush.options.flags = "เพิ่ม flags?"
+; language
+ui.language.title = "เลือกภาษา"
+ui.language.label = "ตั้งค่าภาษาของเซสชั่นของคุณ ถ้าไม่มีภาษาของคุณ คุณสามารถช่วยแปล plugin ได้ที่ GitHub!"
```

Comment:
are this specific terms that cannot be translated?

Reply:
I've updated translation.